docker/ubuntu-ortools

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:latest
+
+# Install requirements
+RUN export DEBIAN_FRONTEND=noninteractive &&\
+    apt-get update -y
+    
+# Install requirements -2
+RUN export DEBIAN_FRONTEND=noninteractive &&\
```

Comment:
consider merging all apt-get commands into one, I expect it will be slightly faster

Reply:
apt-get update is done in one specific RUN because it can be cached. In this case it's faster to change or add some install command after, you don't need to wait for apt-get update when you rebuild your docker image.